src/test/kotlin/com/terraformation/backend/db/DatabaseBackedTest.kt

```diff
@@ -2091,11 +2090,7 @@ abstract class DatabaseBackedTest {
     return rowWithDefaults.id!!
   }
 
-  fun insertTimeZone(timeZone: Any = ZoneId.of("Pacific/Honolulu")): ZoneId {
-    val zoneId = if (timeZone is ZoneId) timeZone else ZoneId.of("$timeZone")
-    timeZonesDao.insert(TimeZonesRow(zoneId))
-    return zoneId
-  }
+  fun insertTimeZone(timeZone: Any = ZoneId.of("Pacific/Honolulu")): ZoneId = ZoneId.of("$timeZone")
```

Comment:
I think it'd be better to update the callers and get rid of this so we aren't left with a misleading function name. The "inline function" refactoring in IntelliJ will do the bulk of the work, so it shouldn't be too painful.